models/llm_layer/llama_layer_prototype.py

```diff
@@ -16,10 +16,7 @@ def call_llama_layer(params="7B", use_cache=False):
     # the models end up with different mlagility hashes
     # Remove the if-statement when
     # https://github.com/groq/mlagility/issues/316 is fixed
```

Comment:
Remove stale comment :)

(these 3 lines, and the 1 line above them)

Reply:
Good catch! Removed!